presto-native-execution/presto_cpp/main/functions/tests/FunctionMetadataTest.cpp

```diff
@@ -40,24 +40,48 @@ class FunctionMetadataTest : public ::testing::Test {
     functionMetadata_ = getFunctionsMetadata();
   }
 
+  static std::string getFunctionKindAndParams(const json& entry) {
```

Comment:
Is this function needed? Could you get the string for `entry["functionKind"]` and `entry["paramTypes"]` in the caller itself with `entry[key].dump()` and hash the strings in place?

Reply:
You're right @pramodsatya , I can call `dump()` method directly. Thanks! I've removed the function.